sites/example-project/src/pages/api/customSettings.json.js

```diff
@@ -1,8 +1,7 @@
 import fs from 'fs';
 import path from 'path';
-import { dev } from '$app/env';
 
-const CUSTOM_SETTINGS_FILE = '.custom-settings.json';
```

Comment:
@mcrascal @hughess In oder for .custom-settings.json to be source controlled, the path has to be like this (i.e so it ends up in the user's root project directory rather than in the template.  This also happens to work nicely in the example project although the the custom settings will be in the mono-repo root.
Having that this feels a bit smelly. If you see any red flags or dealt with a similar with an alternative solution, please share.

Reply:
@ud3sh that path type shows up in a few other places as well (sqlite connector, git file checks) - unfortunately I don't know of an alternative for it